### ISSUE TITLE
BUG: description gets not translated

### ISSUE THREAD

**Issue by cmlnet** [pacollins/hugo-future-imperfect-slim#189]
# Bug Report

## Describe the bug

I noticed that a translation of the _description_ variable gets ignored.

You can observe this on the [live demo](https://themes.gohugo.io//theme/hugo-future-imperfect-slim/fr/).

## To Reproduce

I tried the following two variants in my config.toml:

```
[languages.en]
    LanguageCode        = "en"
    LanguageName        = "English"
    weight = 2
    description = "a hotchpotch"
````

```
[languages.en.params.meta]
    description = "a hotchpotch"
```

## Expected behavior

The description defined under `[params.meta]` should get changed when switching the language.

## Environment

 - **Hugo Version:** [e.g. 0.74.3]

**Comment by pacollins** [comment]
Interesting... I wonder if this is a Hugo thing since the navigation menu _does_ change.

**Comment by cmlnet** [comment]
I was probably partly mistaken. What I wanted to change was `paragraph` not `description`. However, that led me to another curious think: Whenever I added in my config to something like

```
    [languages.en.params.intro]
      paragraph = "a hotchpotch"
```

I had to add all the parameters (e.g. `rssIntro` as well as the whole `params.intro.pic`section) to the language part as the would be otherwise missing in the translated page.
